### ISSUE TITLE
Add a "disabled" step state

### ISSUE THREAD

**Issue by miprib** [sebgroup/ng-wizard#114]
We have a use case where we need to disable a whole step. Refactoring it to sub-steps is not a good solution. In the documentation a disabled step state is shown. However, it is either not supported or not clear how to implement:
![image](https://user-images.githubusercontent.com/32035839/165074323-00b4c399-2d5a-4200-a9a9-80c084768b5e.png)

For our use case, we created a specific CanActivate guard to skip disabled steps. However, it would be nice to have a **visual indication** for this (e.g. like in the picture above).

**Comment by hjalmers** [comment]
Yeah I don't remember why we haven't added a disabled state. Typically we don't disable buttons, links etc. without user interaction i.e. we disable after user interaction together with a toast or alert letting the user know why it's disabled. It would be easy to implement it as a new state, just like `info`, `success` etc. that could be used to mark it as disabled.

I guess using, next and previous actions would need some tweaking too to cater for steps in the middle being disabled. How do you intend to explain why a step is disabled/skipped?

**Comment by miprib** [comment]
Yeah, using next/previous actions (or the left navigation) would require clever CanActivate guards (or another solution). Regarding the explanation, a tooltip (hover on "disabled" state icon) & toast (when the user triggers a "disabled" state)  could be used.
